docs/topics/Gateway_Events.md

```diff
@@ -199,6 +200,27 @@ Due to our privacy and infrastructural concerns with this feature, there are som
 }
 ```
 
+#### Request Soundboard Sounds
+
+Used to request soundboard sounds for a list of guilds. The server will send [Soundboard Sounds](#DOCS_TOPICS_GATEWAY_EVENTS/soundboard-sounds) events for each guild in response.
+
+###### Request Soundboard Sounds Structure
+
+| Field     | Type                | Description                                    | Required |
+|-----------|---------------------|------------------------------------------------|----------|
+| guild_ids | array of snowflakes | IDs of the guilds to get soundboard sounds for | true     |
```

Comment:
Isn't the required column a bit redundant here?

Reply:
fair